.github/workflows/pr-change-set.yaml

```diff
@@ -0,0 +1,21 @@
+# This workflow checks and tests the package code, and it builds all package
+# artifacts whenever there were changes to a pull request.
+
+name: Check change set
+on:
+  pull_request:
+    branches:
+    - main
+    - staging
```

Comment:
As discussed shall we change the target branches to `*` to make sure the checks run on all pull requests?

Reply:
This PR or a separate one?